frontend/www/js/omegaup/components/problem/Tags.vue

```diff
@@ -19,17 +19,21 @@
       <table class="table table-striped">
         <thead>
           <tr>
-            <th class="text-center" scope="col">
+            <th class="text-center" width="50%" scope="col">
```

Comment:
Could it be possible to use a bootstrap-4 class instead? eg. [w-50](https://getbootstrap.com/docs/4.0/utilities/sizing/):
```suggestion
            <th class="text-center w-50" scope="col">
```
?
I think both have the same effect.

It applies in Line 86 too.